ios/FluentUI/Vnext/AvatarGroup/AvatarGroup.swift

```diff
@@ -137,7 +138,10 @@ public struct AvatarGroup: View {
                 let noRingPaddingInterspace = nextAvatarHasRing ? interspace - ringOuterGap : interspace
                 let stackPadding = (currentAvatarHasRing ? ringPaddingInterspace : noRingPaddingInterspace)
 
-                let xOrigin = currentAvatarHasRing ? x + ringOffset : x
+                let xPosition = currentAvatarHasRing ? x + ringOffset : x
+                let xPositionRTL = currentAvatarHasRing ? (nextAvatarHasRing ? -x - ringOuterGap : -x + ringOffset) :
```

Comment:
I'm sure this is right, but I think having three ternary operators in a single line of code _may_ be a bit too much. Can this be broken up at all to be made a bit more legible?